web/src/js/views/series-view.tsx

```diff
@@ -30,20 +30,24 @@ export default class SeriesView extends Component<Props, State> {
         this.seriesChange()
     }
     public async componentDidUpdate(prevProps: Props) {
+<<<<<<< HEAD
```

Comment:
I spy with my little eye something that looks like a merge conflict

Reply:
I forgot to save... woops

Reply:
RIP